### ISSUE TITLE
Need ADB command or Dynamic/Desktop Shortcuts 

### ISSUE THREAD

**Issue by Perpetual-Memories** [Nain57/Smart-AutoClicker#526]
### Describe the feature

[Translate using Tongyi Qianwen]
Add external control capabilities, such as for the app [Extinguish](https://github.com/Moderpach/Extinguish), which allows controlling application services using shell commands.
Integrate specific scripts into the app's dynamic shortcuts or send them to desktop shortcuts, while keeping the status bar tiles to open the most recent scripts.

### Describe the usage

This would enable users to manage the app's state from their computer or integrate with apps like Tasker for automation purposes.
Shortcuts can provide quick access to scripts without needing to enter the app, offering convenience.

### Is it a duplicate?

- [X] There is no duplicates

**Comment by zhrmiah** [comment]
Yes I wa thinking the same. Unfortunately macro is not easy to execute with any kind of quick trigger. Is it possible to integrate this with Tasker? Also if possible adding one click floating shortcut trigger button for macro can help different workflow execution faster.

**Comment by mruac** [comment]
Preferring either intent / adb / Tasker support for me.

Otherwise at the moment I am clicking on the view IDs of the scenario. As long as the scenario name is unique, the view ID will be also.
